contracts/access/GatewayAccessControl.sol

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.16;
+
+import "@openzeppelin/contracts/access/AccessControl.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+bytes32 constant OWNER = keccak256("OWNER");
+bytes32 constant LOW_LEVEL_OPERATOR = keccak256("LOW_LEVEL_OPERATOR");
+bytes32 constant HIGH_LEVEL_OPERATOR = keccak256("HIGH_LEVEL_OPERATOR");
+bytes32 constant FINANCIAL_OWNER = keccak256("FINANCIAL_OWNER");
+bytes32 constant FINANCIAL_OPERATOR = keccak256("FINANCIAL_OPERATOR");
```

Comment:
This could be it's own struct imho

Reply:
An struct can't be immutable or constant, since these values are not bound to change I think we should keep them as constant bytes.

Reply:
I agree with Diego on this one, since this variables will not change I rather keep them as constants instead of a struct that is not immutable.